sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/SaveAsHiveFile.scala

```diff
@@ -124,11 +147,26 @@ private[hive] trait SaveAsHiveFile extends DataWritingCommand {
     val hiveVersion = externalCatalog.unwrapped.asInstanceOf[HiveExternalCatalog].client.version
     val stagingDir = hadoopConf.get("hive.exec.stagingdir", ".hive-staging")
     val scratchDir = hadoopConf.get("hive.exec.scratchdir", "/tmp/hive")
+    logDebug(s"path '${path.toString}' is used")
+    logDebug(s"staging dir '$stagingDir' is used")
+    logDebug(s"scratch dir '$scratchDir' is used")
```

Comment:
Can you pack the three `logDebug`s into one?

Reply:
Yes, I packed these 4 (3+1) logDebug into one.